File: app-ui/MediWeckerUI/Application/Features/Planning/CalendarItem.cs
namespace MediWeckerUI.Application.Features.Planning;

public class CalendarItem
{
    public Medicine Medicine { get; set; }
    public long ScheduledTimeUtc { get; set; }
    public long ActualTimeUtc { get; set; }
    public bool UserResponded { get; set; }
    public long AlarmId { get; set; }
}

File: app/src/main/java/de/htwBerlin/ai/mediAlarm/calendar/data/CalendarItem.kt
package de.htwBerlin.ai.mediAlarm.calendar.data

import de.htwBerlin.ai.mediAlarm.data.medicine.Medicine

data class CalendarItem(
    val medicine: Medicine,
    val alarmId: Long,
    val scheduledTimeUtc: Long,
    val actualTimeUtc: Long,
    val userResponded: Boolean = false
)
